### PR TITLE
LPD-27759

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://liferay.atlassian.net/browse/LPD-27759

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/BaseDDMStructureVersionCTUpgradeProcessTestCase.java [hunk 1]
```diff
@@ -3,24 +3,20 @@
...
-
 /**
- * @author Paulo Albuquerque
+ * @author Carolina Barbosa
  */
-@RunWith(Arquillian.class)
-public class DDMStructureVersionDDMValidationUpgradeProcessTest
-	extends DDMStructureDDMValidationUpgradeProcessTest {
+public abstract class BaseDDMStructureVersionCTUpgradeProcessTestCase
+	extends BaseDDMStructureCTUpgradeProcessTestCase {
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/BaseDDMStructureVersionCTUpgradeProcessTestCase.java:19]: I think it should be BaseDDMStructureVersion**DDMStructure**CTUpgradeProcessTestCase

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v3_8_1/test/DDMStructureVersionCTUpgradeProcessTest.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import com.liferay.dynamic.data.mapping.upgrade.v3_8_1.test.util.DDMFormFieldUpgradeProcessTestUtil;
+
+import org.junit.runner.RunWith;
+
+/**
+ * @author Carolina Barbosa
+ */
+@RunWith(Arquillian.class)
+public class DDMStructureVersionCTUpgradeProcessTest
+	extends BaseDDMStructureVersionCTUpgradeProcessTestCase {
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v3_8_1/test/DDMStructureVersionCTUpgradeProcessTest.java:20]: So here should it be:

```
DDMStructureVersionDDMStructureCTUpgradeProcessTest
	extends BaseDDMStructureVersionDDMStructureCTUpgradeProcessTestCase {
```

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v4_3_1/test/DDMFormInstanceVersionDDMFormInstanceCTUpgradeProcessTest.java [hunk 3]
```diff
@@ -27,8 +27,8 @@
  * @author Paulo Albuquerque
  */
 @RunWith(Arquillian.class)
-public class DDMFormInstanceVersionDDMFormInstanceUpgradeProcessTest
-	extends DDMFormInstanceUpgradeProcessTest {
+public class DDMFormInstanceVersionDDMFormInstanceCTUpgradeProcessTest
+	extends DDMFormInstanceCTUpgradeProcessTest {
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v4_3_1/test/DDMFormInstanceVersionDDMFormInstanceCTUpgradeProcessTest.java:31]: You concatenated it here. That's why I noticed the other example.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_1_0/test/DDMStorageLinkDDMFieldCTUpgradeProcessTest.java [hunk 4]
```diff
@@ -22,8 +22,8 @@
  * @author Paulo Albuquerque
  */
 @RunWith(Arquillian.class)
-public class DDMStorageLinkDDMFieldUpgradeProcessTest
-	extends DDMFieldUpgradeProcessTest {
+public class DDMStorageLinkDDMFieldCTUpgradeProcessTest
+	extends DDMFieldCTUpgradeProcessTest {
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_1_0/test/DDMStorageLinkDDMFieldCTUpgradeProcessTest.java:26]: here too

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_1_1/test/DDMStructureVersionCTUpgradeProcessTest.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+import com.liferay.dynamic.data.mapping.upgrade.v5_1_1.test.util.DDMValidationUpgradeProcessTestUtil;
+
+import org.junit.runner.RunWith;
+
+/**
+ * @author Paulo Albuquerque
+ */
+@RunWith(Arquillian.class)
+public class DDMStructureVersionCTUpgradeProcessTest
+	extends BaseDDMStructureVersionCTUpgradeProcessTestCase {
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_1_1/test/DDMStructureVersionCTUpgradeProcessTest.java:20]: Apply the changes here: 
```
DDMStructureVersionDDMStructureCTUpgradeProcessTest
	extends BaseDDMStructureVersionDDMStructureCTUpgradeProcessTestCase {
```

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_3_2/test/DDMFormInstanceVersionDDMFormInstanceCTUpgradeProcessTest.java [hunk 6]
```diff
@@ -21,8 +21,8 @@
  * @author Paulo Albuquerque
  */
 @RunWith(Arquillian.class)
-public class DDMFormInstanceVersionDDMFormInstanceSettingsUpgradeProcessTest
-	extends DDMFormInstanceDDMFormInstanceSettingsUpgradeProcessTest {
+public class DDMFormInstanceVersionDDMFormInstanceCTUpgradeProcessTest
+	extends DDMFormInstanceCTUpgradeProcessTest {
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_3_2/test/DDMFormInstanceVersionDDMFormInstanceCTUpgradeProcessTest.java:25]: you did it here too.

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**